packages/amazon-ssm-agent/amazon-ssm-agent.spec

```diff
@@ -17,37 +12,73 @@ Source0: %{gorepo}-%{version}.tar.gz
 Source1000: clarify.toml
 
 BuildRequires: %{_cross_os}glibc-devel
+Requires: %{name}(binaries)
```

Comment:
Each top-level package now depends on a capability like this to ensure that its binaries are installed - since the package wouldn't be very useful without those!